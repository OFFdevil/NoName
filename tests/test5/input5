main // bad
  (A)&&(B);    // bad
 // bad
  (A)^((struct A));