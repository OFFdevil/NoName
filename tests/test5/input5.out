Syntax error LexToken(DIVIDE,'/',1,5)
Syntax error LexToken(SPACE,' ',4,43)
